Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -615,6 +627,28 @@ protected override void RenderContents (HtmlTextWriter writer)
         menu.RenderAsContextMenu(writer);
     }
 
+    private void RenderAriaTreeAlertBox (HtmlTextWriter writer)
+    {
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaLive, HtmlAriaLiveAttributeValue.Assertive);
+      writer.RenderBeginTag(HtmlTextWriterTag.Span);
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaHidden, HtmlAriaHiddenAttributeValue.True);
+      writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassScreenReaderText + " AriaTreeAlerts");
+      writer.RenderBeginTag(HtmlTextWriterTag.Span);
+      writer.Write("&nbsp;");
+      writer.RenderEndTag();
+      writer.RenderEndTag();
+    }
+
+    private void RenderAriaPossibleChildrenDescription (HtmlTextWriter writer)
+    {
+      writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassScreenReaderText);
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaHidden, HtmlAriaHiddenAttributeValue.True);
+      writer.AddAttribute(HtmlTextWriterAttribute.Id, $"{c_possibleChildrenDescriptorIDPrefix}_{ID}");
+      writer.RenderBeginTag(HtmlTextWriterTag.Span);
+      writer.Write(GetResourceManager().GetString(ResourceIdentifier.ScreenReaderPossiblyNoChildrenText));
```

Comment:
we got better alternatives:
`GetResourceManager.GetText(...).WriteTo(writer)`
This ensures html encoding is done properly.